actions/rules/action_state_to_metastate.go

```diff
@@ -0,0 +1,416 @@
+package rules
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+
+	uuid "github.com/satori/go.uuid"
+
+	"github.com/fabric8-services/fabric8-wit/application"
+	"github.com/fabric8-services/fabric8-wit/convert"
+	"github.com/fabric8-services/fabric8-wit/workitem"
+)
+
+type ActionStateToMetaState struct {
+	Db     application.DB
+	Ctx    context.Context
+	UserID *uuid.UUID
+}
+
+// make sure the rule is implementing the interface.
+var _ Action = ActionStateToMetaState{}
+
+func (act ActionStateToMetaState) contains(s []uuid.UUID, e uuid.UUID) bool {
```

Comment:
Can you outsource this function into the `id` package please? We already have utility functions there and this would fit quite well.

Reply:
Outdated. Already fixed. This is not using UUIDs anymore.